src/assets/icon/CommentReply.js

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import Svg, {Path} from 'react-native-svg';
-import {COLORS} from '../../utils/theme';
+import {colors} from '../../utils/colors';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to use a lowercase `colors` instead of the previous `COLORS`. This change seems fine as long as the exported object from `'../../utils/colors'` is indeed named `colors`. If not, this will cause an error. Please ensure that the exported object from `'../../utils/colors'` matches the imported name here.

```diff
- import {COLORS} from '../../utils/theme';
+ import {colors} from '../../utils/colors';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->